actions/images_action.js

```diff
@@ -0,0 +1,272 @@
+                url: URL + '/api/photos/delete',
+                method: 'DELETE',
+                headers: {
+                    Authorization: `Bearer ${token}`,
+                    'Content-Type': 'application/json'
+                },
+                params: { photoId }
+            });
+        } catch (error) {
+            console.log('delete web image', error);
+        }
+        console.log(response.data);
+        if (response && response?.data?.success) {
+            dispatch({
+                type: DELETE_IMAGE,
+                payload: id
+            });
+        }
+    };
+};
+
+/**
+ * Increment the amount of photos selected for deletion
+ */
+export const incrementSelected = () => {
+    return {
+        type: INCREMENT_SELECTED
+    };
+};
+
+/**
+ * remove a tag from image
+ */
+export const removeTagFromImage = (data) => {
+    return {
+        type: REMOVE_TAG_FROM_IMAGE,
+        payload: data
+    };
+};
+
+/**
+ * Toggle if the user wants to Select images for deletion
+ */
+export const toggleSelecting = () => {
+    return {
+        type: TOGGLE_SELECTING
+    };
+};
+
+/**
+ * toggle selected property of a image object
+ * @param {number} id
+ */
+export const toggleSelectedImage = (id) => {
+    return {
+        type: TOGGLE_SELECTED_IMAGES,
+        payload: id
+    };
+};
+
+/**
+ * fn to upload images along with tags
+ * @param {string} token
+ * @param  image form data
+ * @returns
+ */
+export const uploadImage = (token, image, imageId) => {
+    let response;
+    return async dispatch => {
+        try {
+            response = await axios(URL + '/api/photos/upload-with-tags', {
+                method: 'POST',
+                headers: {
+                    Authorization: 'Bearer ' + token,
+                    'Content-Type': 'multipart/form-data'
+                },
+                data: image
+            });
+        } catch (error) {
+            if (error.response) {
+                console.log(
+                    'ERROR: shared_actions.upload_photo',
+                    JSON.stringify(error?.response?.data, null, 2)
+                );
+            } else {
+                // Other errors -- NETWORK ERROR
+                console.log(error);
+            }
+
+            return {
+                success: false
+            };
+        }
+        console.log('Response: shared_actions.uploadPhoto', response?.data);
+
+        if (response && response.data?.success) {
+            dispatch({
+                type: DELETE_IMAGE,
+                payload: imageId
+            });
+            // return the photo.id that has been created on the backend
+            return {
+                success: true,
+                photo_id: response.data.photo_id
+            };
+        }
+    };
+};
+
+/**
+ * Upload the tags that were applied to an image
+ * @param {string} token
+ * @param  image - the image object
+ */
+export const uploadTags = (token, image) => {
```

Comment:
This is now for web image only? Function name or comment needs to be updated to explain this to the reader

Reply:
Yes now its for web image only... I will add comments